moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryStatusUpdate.js

```diff
@@ -141,6 +143,12 @@ const ProjectSummaryStatusUpdate = ({ projectId, data, refetch, classes }) => {
     setStatusUpdateAddNew(false);
   };
 
+  /**
+   * Only allow the user who wrote the status to edit it
+   */
+  const editableComment =
+    userSessionData.user_id === parseInt(getStatusUpdate("added_by_user_id"));
```

Comment:
can you use the `userId` variable you declared above here too?

Reply:
yes, thanks for pointing that out!